agent_params_loader:
   ros__parameters:
       sampling_time: 0.033
       runge_kutta_45: True
       ground_height: -0.1
       robot_control: True
       robot_name: "pioneer3at"
       laser_sensor_name: "laser"
       robot_radius: 0.3
       robot_mass: 35
       robot_velocity: 0.9
       robot_relaxation_time: 0.5
       robot_k_orthogonal: 1.0
       robot_k_damping: 500.0
       robot_k_lambda: 0.3
       robot_alpha: 3.0
       robot_Ai: 2000.0
       robot_Aw: 2000.0
       robot_Bi: 0.08
       robot_Bw: 0.08
       robot_k1: 120000.0
       robot_k2: 240000.0
       robot_initial_orientation: -180.0
       robot_waypoints: [-6.5,-0.0,6.5,0.0]
       robot_initial_position: [6.5,0.0]
       robot_ignore_obstacles: ["circular_workspace"]
       attach_cylinder_to_humans: True
       consider_robot: True
       agents: ["agent1","agent2","agent3","agent4","agent5"]
       agent1:
           waypoints: [6.5,0.0082,-6.5,-0.0082]
           initial_orientation: 0.0724
           initial_position: [-6.5,-0.0082]
           radius: 0.3
           mass: 75
           relaxation_time: 0.5
           k_orthogonal: 1.0
           k_damping: 500.0
           k_lambda: 0.3
           alpha: 3.0
           group_distance_forward: 2.0
           group_distance_orthogonal: 1.0
           k1g: 200.0
           k2g: 200.0
           Ai: 2000.0
           Aw: 2000.0
           Bi: 0.08
           Bw: 0.08
           k1: 120000.0
           k2: 240000.0
           velocity: 0.9
           animation_factor: 5.1
           animation_name: "walking"
           people_distance: 6.0
           ignore_obstacles: ["circular_workspace"]
           publish_forces: False
       agent2:
           waypoints: [-0.9376,6.432,0.9376,-6.432]
           initial_orientation: 98.2938
           initial_position: [0.9376,-6.432]
           radius: 0.3
           mass: 75
           relaxation_time: 0.5
           k_orthogonal: 1.0
           k_damping: 500.0
           k_lambda: 0.3
           alpha: 3.0
           group_distance_forward: 2.0
           group_distance_orthogonal: 1.0
           k1g: 200.0
           k2g: 200.0
           Ai: 2000.0
           Aw: 2000.0
           Bi: 0.08
           Bw: 0.08
           k1: 120000.0
           k2: 240000.0
           velocity: 0.9
           animation_factor: 5.1
           animation_name: "walking"
           people_distance: 6.0
           ignore_obstacles: ["circular_workspace"]
           publish_forces: False
       agent3:
           waypoints: [0.9132,6.4355,-0.9132,-6.4355]
           initial_orientation: 81.9236
           initial_position: [-0.9132,-6.4355]
           radius: 0.3
           mass: 75
           relaxation_time: 0.5
           k_orthogonal: 1.0
           k_damping: 500.0
           k_lambda: 0.3
           alpha: 3.0
           group_distance_forward: 2.0
           group_distance_orthogonal: 1.0
           k1g: 200.0
           k2g: 200.0
           Ai: 2000.0
           Aw: 2000.0
           Bi: 0.08
           Bw: 0.08
           k1: 120000.0
           k2: 240000.0
           velocity: 0.9
           animation_factor: 5.1
           animation_name: "walking"
           people_distance: 6.0
           ignore_obstacles: ["circular_workspace"]
           publish_forces: False
       agent4:
           waypoints: [1.8201,6.24,-1.8201,-6.24]
           initial_orientation: 73.7385
           initial_position: [-1.8201,-6.24]
           radius: 0.3
           mass: 75
           relaxation_time: 0.5
           k_orthogonal: 1.0
           k_damping: 500.0
           k_lambda: 0.3
           alpha: 3.0
           group_distance_forward: 2.0
           group_distance_orthogonal: 1.0
           k1g: 200.0
           k2g: 200.0
           Ai: 2000.0
           Aw: 2000.0
           Bi: 0.08
           Bw: 0.08
           k1: 120000.0
           k2: 240000.0
           velocity: 0.9
           animation_factor: 5.1
           animation_name: "walking"
           people_distance: 6.0
           ignore_obstacles: ["circular_workspace"]
           publish_forces: False
       agent5:
           waypoints: [-2.6975,-5.9138,2.6975,5.9138]
           initial_orientation: -114.5191
           initial_position: [2.6975,5.9138]
           radius: 0.3
           mass: 75
           relaxation_time: 0.5
           k_orthogonal: 1.0
           k_damping: 500.0
           k_lambda: 0.3
           alpha: 3.0
           group_distance_forward: 2.0
           group_distance_orthogonal: 1.0
           k1g: 200.0
           k2g: 200.0
           Ai: 2000.0
           Aw: 2000.0
           Bi: 0.08
           Bw: 0.08
           k1: 120000.0
           k2: 240000.0
           velocity: 0.9
           animation_factor: 5.1
           animation_name: "walking"
           people_distance: 6.0
           ignore_obstacles: ["circular_workspace"]
           publish_forces: False
